pull-request-template.md

```diff
@@ -0,0 +1,8 @@
+#### What does this PR do?
+
+
+#### Related JIRA tickets
+
+
+#### Includes new or updated dependencies?
```

Comment:
@ehanson8 Not entirely clear on this. This should be probably be captured in the first section?

Reply:
Keeping after webex chat